spec/system/visit_users_spec.rb

```diff
@@ -13,9 +13,15 @@
   let(:team_cherry) { create(:new_team_event) }
   let(:team_rp) { create(:new_team_event, team_type: 'rp') }
   let(:team_idp) { create(:new_team_event, team_type: 'idp') }
+  let(:team_other) { create(:new_team_event, team_type: 'other') }
+
   let(:create_teams) { team_apple
                         team_banana
-                        team_cherry }
+                        team_cherry
+                        team_rp
+                        team_idp
+                        team_other
+                      }
```

Comment:
Try and put `team_apple` on the next line, so the teams all indent the same way and put a line at the end of  `}`